app/helpers/sessions_helper.rb

```diff
@@ -11,6 +11,10 @@ def remember user
     cookies_permanent[:remember_token] = @current_user.remember_token
   end
 
+  def current_user? user
```

Comment:
mục đích của hàm này để làm gì?

Reply:
hàm này kiểm tra xem user đang dùng và user đã đăng nhập có giống nhau hay không.

Reply:
việc này là check logic sao lại cho vào helper?
mục đích của helper là gi?

Reply:
Khi các xử lí logic phía model quá nhiều, làm model bị phình to, ta có thể sử dụng helper để định nghĩa các method, các method này có thể được gọi trực tiếp từ view. 
Mục đích của helper là để hỗ trợ cho view.

Reply:
Vâng, em chuyển hàm curent_user vào model rồi ạ